### PR TITLE
Closes # 48 Update NEWS.md ahead of first release

### LINKED ISSUE #48: Documentation:  Update `NEWS.md`  ahead of first release
### Please select a category the issue is focused on?

_No response_

### Let us know where something needs a refresh or put your idea here!

The `NEWS.md` file populates the changelog tab on the admiralmetabolic website. It usually contains updates on what's changed during successive releases of the package. See for example the [{admiral} changelog](https://pharmaverse.github.io/admiral/news/index.html). However, when you are working towards the first release, there is no need to populate `NEWS.md` as everything is new.  Currently, our `NEWS.md` is just the default one coming from {admiraltemplate}/

We need to update `NEWS.md` so that it indicates that {admiralmetabolic} 0.1.0 will be the first release of the package. See for example the [{admiralpeds}](https://pharmaverse.github.io/admiralpeds/news/index.html) `NEWS.md`:

image

### PR DESCRIPTION
**Pull request by kiwi5104679**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #48 into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by kiwi5104679** [comment]
Hi @manciniedoardo.
devtools::check() and pkgdown::build_site() do not work again.
The cause is that the function derive_vars_cat() is used but cannot be found.
In my understanding, the function derive_vars_cat() does not exist in latest version of {admiral} so far.
image

Is there any way to remove this error?

File: NEWS.md [hunk 1]
```diff
@@ -1,8 +1,4 @@
 # admiralmetabolic 0.1.0
 
-- Initial package release focused on `<<ADD INITIAL SCOPE>>`
-
-Note that after first release you should then start maintaining this `NEWS.md` file for every user-facing update.
-You should categorize the updates under title sections for New Features, Updates of Existing Functions,
-Breaking Changes, Documentation, Various.
-Each update should also reference the original issue by adding at the end: (#ISSUE-NUMBER)
+- Initial package release mainly focused on obesity indicators
```
Comment (manciniedoardo) [inline NEWS.md:3]:
```suggestion
- Initial package release mainly focused on obesity therapeutic area
```

File: NEWS.md [hunk 2]
```diff
@@ -1,8 +1,4 @@
...
+(e.g. Waist to Hip Ratio, Waist to Height Ratio).
```
Comment (manciniedoardo) [inline NEWS.md:4]:
```suggestion
(e.g. functions for Waist to Hip Ratio, Waist to Height Ratio).
```

Comment (manciniedoardo) [reply NEWS.md:4]: I think if we are going to mention the functions then we should mention our templates and vignettes too. Alternatively as this is our first release we could omit mentioning anything like that in the changelog at all. @AndersAskeland what do you think?

**Review by manciniedoardo — COMMENTED** [review]
Some initial comments - thanks @kiwi5104679

File: NEWS.md (hunk 2 above)
Comment (manciniedoardo) [reply NEWS.md:4]: ok, happy to go with this

Comment (AndersAskeland) [reply NEWS.md:4]: @kiwi5104679 What are your thoughts on this?

Comment (kiwi5104679) [reply NEWS.md:4]: @AndersAskeland , @manciniedoardo , thank you.
I think it's better to have subsections.
I'll add "Functions" and "Documentation".

Comment (kiwi5104679) [reply NEWS.md]: Hi @AndersAskeland , @manciniedoardo.
I've updated NEWS.md.
I noticed that {admiral} and other related packages use "New Features" subsection to describe new functions instead of "Functions", so I added "New Features" subsection in {admiralmetabolic}.
Please let me know if it's better to use "Functions".
image
image

Comment (kiwi5104679) [reply NEWS.md]: In addition, should we have hyperlinks in changelog?
image

It's user friendly to have hyperlinks, but I'm concerned that maintenance may be difficult.

Comment (kiwi5104679) [reply NEWS.md]: Exactly.
Updated subsection to "Functions".

File: NEWS.md [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
+(e.g. functions for Waist to Hip Ratio, Waist to Height Ratio).
 
-Note that after first release you should then start maintaining this `NEWS.md` file for every user-facing update.
-You should categorize the updates under title sections for New Features, Updates of Existing Functions,
-Breaking Changes, Documentation, Various.
-Each update should also reference the original issue by adding at the end: (#ISSUE-NUMBER)
+## Functions
+
+- The new function "'derive_param_waisthip'" is provided to derive parameters for Waist to Hip Ratio. (#33 &  #52)
+- The new function "'derive_param_waisthgt'" is provided to derive parameters for Waist to Height Ratio. (#33 & #52)
```
Comment (AndersAskeland) [inline NEWS.md:9]:
```suggestion
- The function `derive_param_waisthip()` for deriving Waist to Hip Ratio. (#33 &  #52)
- The function `derive_param_waisthgt()` for deriving Waist to Height Ratio. (#33 & #52)
```

File: NEWS.md [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+
+## Documentation
+
+- New vignette "Creating a Metabolic ADVS ADaM" has been created. (#28) 
+- New vignette "Creating a Control of Eating Questionnaire ADaM" has been created. (#53)
+- New template scripts for ADVS "'ad_advs.R'" has been created. (#35)
```
Comment (AndersAskeland) [inline NEWS.md:15]:
```suggestion
- Vignette "Creating a Metabolic ADVS ADaM". (#28) 
- Vignette "Creating a Control of Eating Questionnaire ADaM". (#53)
- ADVS template scripts `ad_advs.R` which creates a Vital Sign Analysis Dataset with metabolic specific derivations. (#35)
```

**Review by AndersAskeland — CHANGES_REQUESTED** [review]
Great work @kiwi5104679 ! I have a couple of minor text change requests :)

File: NEWS.md (hunk 4 above)
Comment (kiwi5104679) [reply NEWS.md]: Hi @AndersAskeland , I updated quotations and wordings.
Could you take a look again?

File: NEWS.md [hunk 5]
```diff
@@ -1,8 +1,15 @@
 # admiralmetabolic 0.1.0
 
-- Initial package release focused on `<<ADD INITIAL SCOPE>>`
+- Initial package release mainly focused on obesity therapeutic area
+(e.g. functions for Waist to Hip Ratio, Waist to Height Ratio).
```
[the remark this answers is not in the document]
Comment (kiwi5104679) [reply NEWS.md]: Deleted example.

**Comment by kiwi5104679** [comment]
Hi @AndersAskeland , thank you for your comments.
I updated quotations and wordings.
Could you take a look again?

File: NEWS.md (hunk 1 above)
Comment (AndersAskeland) [reply NEWS.md]: This appears resolved :)

File: NEWS.md [hunk 6]
```diff
@@ -1,8 +1,14 @@
 # admiralmetabolic 0.1.0
 
-- Initial package release focused on `<<ADD INITIAL SCOPE>>`
+- Initial package release mainly focused on obesity therapeutic area
```
Comment (AndersAskeland) [inline NEWS.md:3]:
```suggestion
- Initial package release mainly focused on obesity therapeutic area.
```

**Review by AndersAskeland — CHANGES_REQUESTED** [review]
Great work @kiwi5104679 . I overlooked the period mark last review, but besides that I think we should merge. You have any other comments @manciniedoardo ?

**Comment by kiwi5104679** [comment]
@AndersAskeland , thank you for your comment.
Added the period mark.

**Review by AndersAskeland — APPROVED** [review]
LGTM! Again, thanks for tasking this @kiwi5104679 👍

**Comment by AndersAskeland** [comment]
I think it is a good idea to update the DESCRIPTION file as proposed by @yurovska . Would you be able to do that @kiwi5104679 ?

**Comment by kiwi5104679** [comment]
Hi @yurovska , @AndersAskeland , thank you for your comments!
Sure. I take it.

**Comment by AndersAskeland** [comment]
Great @kiwi5104679 ! Do you have time to add it before our meeting? If not, I can also add it tomorrow morning Danish time 👍

**Comment by kiwi5104679** [comment]
Hi @AndersAskeland , sorry for delay.
I've just changed the version in DISCRIPTION and pushed it!
Please review and approve.
